runtime/jsMain/src/kotlinx/benchmark/js/JsBuiltInExecutor.kt

```diff
@@ -0,0 +1,49 @@
+package kotlinx.benchmark.js
+
+import kotlinx.benchmark.*
+import kotlinx.benchmark.jsEngineSupport
+
+@JsName("Function")
+private external fun functionCtor(params: String, code: String): (dynamic) -> Long
+
+class JsBuiltInExecutor(
+    name: String,
+    @Suppress("UNUSED_PARAMETER") dummy_args: Array<out String>
+) : CommonSuiteExecutor(name, jsEngineSupport.arguments()[0]) {
+
+    private val BenchmarkConfiguration.jsUseBridge: Boolean
+        get() = "true".equals(advanced["jsUseBridge"], ignoreCase = true)
```

Comment:
I am not sure that this feature should be disclosed to the users. These bridges are quite fragile. Maybe it would be even better to make a gradle console warning and not to document it. What do you think?

Reply:
Currently, I am not sure what is the purpose of this flag. How does it affect the runtime? And on what occasions it is supposed to be used?

Reply:
This flag needed to enable feature that create special wrappers. These wrappers are prevents inlining optimisations in JS and Wasm while using their builtin benchmark executors. This feature uses a little hacky and possibly unstable technics so it is disabled by default. (and maybe it should not be discoloured to public in docs).

Hopefully this technics will disable inliner to inject payload into the benchmark runtime and keep it isolated.
For `js` it creates special wrapper that create new js function for measure on each call of measurer - the measurer function will be new on each call (AFAIK BenchmarkJS do the same for that).
For `wasm` it just pass the wasm measurer via the `js world` - it works because of the lack of crossinliner for wasm-js worlds.

It needed to prevent inlining optimisations of the payload with benchmark runtime. For wasm it is really needed because such optimisation could lead to incorrect benchmarking results and even huge performance degradation of measured code.